taca/cli.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import click
+import taca.log
+
+from pkg_resources import iter_entry_points
+
+from taca.utils import config
+
+
+@click.group()
```

Comment:
I can show you one new thing! Click officially supports displaying version information:

But you probably need to follow some convention of putting the version in your root ``__init__.py`` to make it automated:

**__init__.py**
```python
__version__ = '0.1.1'
```

**cli.py**
```python
from taca import __version__

@click.group()
@click.version_option(__version__)
```

Then Click sets the rest up for you:

```bash
$ taca --version
taca, version 0.1.1
```